File: kods_C_C++/PYTHON/mario.py
import cs50

from cs50 import get_int


def main():

    height = get_height()
    for i in range(height):
        print("#",end = "")
    print()


def get_height():
    while True:
        # height = get_int("Height: ")
        try:
            height = int(input("Height: "))
            if height > 0 & height < 9:
                break
        except ValueError:
            print("Not an int")

    return height


main()

File: kods_C_C++/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{

    for (int i = 0; i < 4; i++)
    {
        printf("?\n");
    }
    printf("\n");

    const int square = 5;
    for (int i = 0; i < square; i++)
    {
        for (int j = 0; j < square; j++)
        {
            printf("X ");
        }
        printf("\n");
    }

    int num;

    do
    {
        num = get_int("Size? \n");
    } while (num < 1);

    for (int i = 0; i < num; i++)
    {
        for (int j = 0; j < num; j++)
        {
            printf("X ");
        }
        printf("\n");
    }
}